sdks/python/apache_beam/coders/coder_impl.py

```diff
@@ -570,6 +571,15 @@ def _construct_from_sequence(self, components):
 class WindowedValueCoderImpl(StreamCoderImpl):
   """A coder for windowed values."""
 
+  # TODO: Fn Harness only supports millis. Is this important enough to fix?
+  def _to_normal_time(self, value):
+    """Convert "lexicographically ordered unsigned" to signed."""
```

Comment:
Can you document what "normal time" means for coders?  What is "lexicographically ordered unsigned"?